docs/sdks/source-sdk/objects.md

```diff
@@ -7,6 +7,7 @@ Currently the SDK defines 5 types of objects:
 * Threads: provide a way to structure the messages. Examples: blog posts, forum threads. Not mandatory (eg. Twitter).
 * Users: every other object type must have an author. This is the only object type that has no actions available.
 * Status: provides a way to update messages and private messages, like marking them as read. Only used in [Send API](../send-api)
+* Typing: provides a way of notifying Engage Digital that an end-user is currently typing a message. Only used in [Send API](../send-api)
```

Comment:
You should also change "Currently the SDK defines 5 types of objects" to "Currently the SDK defines 6 types of objects"

Reply:
🙈 good catch, done c455295